bot/cogs/snakes.py

```diff
@@ -41,7 +127,111 @@ def __init__(self, bot: AutoShardedBot):
         :param name: Optional, the name of the snake to get information for - omit for a random snake
         """
 
+        name = name.lower()
+        dprint(name)
+
+        custom = False
+
+        if name == 'python':
+            rating = '🐍'
+            common = 'Python'
+            length = '88.1 MB'
+            uimage = 'https://www.python.org/static/img/python-logo.png'
+            spit = 'venomous'
+            scient = 'cpython/master'
+            custom = True
+
+        elif name == 'anaconda':
+            rating = '---'
+            common = 'Anaconda Cloud'
+            length = '???'
+            uimage = 'https://upload.wikimedia.org/wikipedia/en/thumb/c/cd/Anaconda_Logo.png/' \
+            '200px-Anaconda_Logo.png'
+            spit = 'none'
+            scient = 'anaconda-project/master'
+            custom = True
+
+        elif name == 'electron':
+            dprint(name, '==electron')
+            rating = '🐍🐍🐍🐍🐍'
+            spit = 'extremely venomous'
+            common = 'Pure Evil'
+            uimage = "https://vignette.wikia.nocookie.net/happytreefanon/images/9/95/" \
+            "Tumblr_m2g660GN9z1qcwgmzo1_1280.png/revision/latest?cb=20120611212456"
+            scient = 'Chironex fleckeri'
+            length = '9.8 ft'
+            custom = True
```

Comment:
These 3 sneks fall under the same issue as the danger dict. Move this data into a json file!